README.md

```diff
@@ -51,15 +42,28 @@ BLT provides an automation layer for testing, building, and launching Drupal 8 a
     ```
     $ composer install
     ```
-6. Route the site of your choice in `sites.local.php`.
-   ```
-   $sites['8888.localhost'] = 'mysite';
-   ```
-7. Configure `local.settings.php` to override Lando defaults.
-8. Start the built-in PHP server.
+6. Create all `sites/*/settings/local.settings.php` files.
+    ```
+    blt blt:init:settings
+    ```
+7. Configure each `sites/*/settings/local.settings.php` file to connect to the appropriate MySQL database. You'll need to use the user/password you set during step 3. The host should `localhost`.
+8. If the databases do not exist, use `drush sql:create` to create them. For example:
+    ```
+    drush -l mysite sql:create
+    ```
+9. Start the built-in PHP server.
```

Comment:
Assumes drush is installed globally and Acquia Dev Desktop is not doing shady things in your bash_profile/bashrc